_includes/nav.html

```diff
@@ -9,7 +9,7 @@
   }
 </style>
 
-<header class="header" data-swiftype-index="false">
```

Comment:
Is there a reason you're removing this?

Reply:
I didn't intend to remove this. Probably, removed due to merge conflict resolution. Hence, brought back